packages/opentelemetry/src/main.ts

```diff
@@ -0,0 +1,67 @@
+import { EventStoreDBInstrumentation } from ".";
+
+import {
+  SimpleSpanProcessor,
+  BasicTracerProvider,
+  InMemorySpanExporter,
+} from "@opentelemetry/sdk-trace-node";
+import { context } from "@opentelemetry/api";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { v4 } from "uuid";
+import { AsyncHooksContextManager } from "@opentelemetry/context-async-hooks";
+
+const contextManager = new AsyncHooksContextManager().enable();
+
+context.setGlobalContextManager(contextManager);
+
+const inMemorySpanExporter = new InMemorySpanExporter();
+
+const provider = new BasicTracerProvider();
+
+provider.addSpanProcessor(new SimpleSpanProcessor(inMemorySpanExporter));
```

Comment:
Is this file for testing purposes? It shouldn't be necessary for instrumented code to reference the OpenTelemetry SDK and or spin up instances of Otel

Reply:
Removed 👍